core/src/main/java/com/epam/coroutinecache/core/Memory.kt

```diff
@@ -8,15 +8,15 @@ interface Memory {
      * @param key The key associated with the Record to be retrieved from memory
      * @see Record
      */
-    fun <T> getRecord(key: String): Record<T>?
+    fun <T: Any> getRecord(key: String): Record<T>?
```

Comment:
what is the point of extending Any?

Reply:
> what is the point of extending Any?

My bad, extending Any came from my playground